### PR TITLE
Add outliers and button

### LINKED ISSUE #9: Add outlier tagging for cash bail race table
We should automatically hide outlier counties, and only show their rows when the user presses the "show outliers" button from the design.

### PR DESCRIPTION
**Pull request by alexwgraves**
Closes #9 

Once you've merged #5 into master, I'll merge master into this and refine the styles a bit so it matches the designs. (This "Show Outliers" button is supposed to go inline with the search.)

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -88,8 +88,14 @@ const createBailRaceTable = () => {
...
-createBailRateTable();
-createBailRaceTable();
+/* RENDER PAGE */
+const bailRateTable = createBailRateTable();
+
+const bailRaceTable = createBailRaceTable();
+document.getElementById("race-outliers").addEventListener("click", (e) => {
+  const showOutliers = bailRaceTable.toggleOutliers();
+  e.target.className = showOutliers ? "outliers-btn showing" : "outliers-btn";
+});
```
Comment (apothuku) [inline src/js/index.js:111]: How would you like to handle these? Leave them outside of the Table class? Create a TableContainer wrapper class, and then the TableContainer would have a reference to the search, outlier, and viewall buttons? Would the TableContainer handle the event listeners?

**Review by apothuku — COMMENTED** [review]
This is a bit of a messy merge with search, so allow me to make a first pass of merging master into this.

File: src/js/classes/Table.js [hunk 2]
```diff
@@ -307,6 +311,36 @@ export class RankedTable {
     }
   }
 
+  initRender() {
```
Comment (apothuku) [inline src/js/classes/Table.js]: Is there a better name for this? Perhaps something that follows convention?

**Review by apothuku — COMMENTED** [review]
Question: If "show outliers" is disabled, but I search for an outlier county, should the county stil show up? Currently it does not - should we always show the result of the search no matter the other filters? Something to ask Rong perhaps?

File: src/js/index.js (hunk 1 above)
Comment (alexwgraves) [reply src/js/index.js:111]: Eh, let's leave them out. It's just five lines. (And only some of the tables have them.)

File: src/js/classes/Table.js (hunk 2 above)
Comment (alexwgraves) [reply src/js/classes/Table.js]: Maybe just `init`? I don't have strong feelings.

**Review by alexwgraves — COMMENTED** [review]
Yeah, let's ask Rong about that.

**Comment by alexwgraves** [comment]
From Rong:
> I think whenever you’re in search mode you can access all counties, and say if you select an outlier county, then that would turn the ‘show outliers’ toggle on if it was off before

**Pull request merged**